### PR TITLE
Add suffixes to typos that start with "as"

### PR DESCRIPTION
**Pull request by int-y1**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -5279,50 +5353,101 @@ assignenmet->assignment
...
 assignmnet->assignment
+assignmnets->assignments
 assignt->assign
+assignted->assigned
+assignting->assigning
 assigntment->assignment
+assigntments->assignments
+assignts->assigns
+assihn->assign
+assihned->assigned
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:5371]: Perhaps add `assignee`/`assignees` typos, unless they are not frequent enough.

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -5279,50 +5353,101 @@ assignenmet->assignment
...
+assiociations->associations
+assiociative->associative
 assisance->assistance
 assisant->assistant
 assisants->assistants
 assising->assisting
 assisnate->assassinate
 assisst->assist, assists,
+assisstance->assistance
+assisstant->assistant
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:5428]: What about adding typos of `assistive`?

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -5279,50 +5353,101 @@ assignenmet->assignment
...
+assisstants->assistants
+assissted->assisted
+assissting->assisting
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:5431]: What about adding typos of `assistive`? You've already added a typo above, but about other typos such as `assisstive`?

**Review by DimitriPapadopoulos — APPROVED** [review]

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -5279,50 +5355,107 @@ assignenmet->assignment
...
+assimes->assumes
+assiming->assuming
+assimption->assumption
+assimptions->assumptions
 assimtote->asymptote
 assimtotes->asymptotes
 assined->assigned
+assinee->assignee
+assinees->assignees
 assing->assign
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:5393]: By the way, `assing` could also be `gassing`, `massing`, `passing`, `sassing`. Maybe `assessing` too. Any such typos out there?

Comment (int-y1) [reply codespell_lib/data/dictionary.txt:5393]: I couldn't find any of these.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**